### DISCUSSION TITLE
Build / compile instructions?

### DISCUSSION THREAD

**Discussion by tgrushka** [CarlGao4/Demucs-Gui#64] [Q&A]
How do I build this package myself on M1 macOS Sonoma? We can better help you fix bugs and test models if we can build this package ourselves.

**Comment by CarlGao4** [comment]
It's quite complicated, though. I think we needn't to do this, as you can just open an issue if you want a new feature, or run the codes yourself and I'll find a solution to pack it.

**Reply by CarlGao4** [reply]
In fact, it takes me about 80 minutes to pack a version each time
